### ISSUE TITLE
Better Werkzeug Support

### ISSUE THREAD

**Issue by staab** [rollbar/pyrollbar#229]
I realize this is my fault for using Werkzeug in my application directly, so feel free to close this if first-class werkzeug support isn't a priority for rollbar. 

It was pretty painful to get requests to get sent to rollbar when using werkzeug (to the point where I had to monkey-patch rollbar to get things working), so I thought I'd document here how I got it working.

The main problem was that I subclassed the werkzeug BaseRequest (as recommended on line 111 [here](https://github.com/pallets/werkzeug/blob/master/werkzeug/wrappers.py)) in order to use some of their extra request wrappers:

```
from werkzeug.wrappers import (
    BaseRequest,
    AuthorizationMixin,
    CommonRequestDescriptorsMixin)
from werkzeug.contrib.wrappers import (
    JSONRequestMixin, RoutingArgsRequestMixin)


class Request(
        BaseRequest,
        AuthorizationMixin,
        CommonRequestDescriptorsMixin,
        JSONRequestMixin,
        RoutingArgsRequestMixin):
    pass
```

This doesn't work with rollbar, because of line 64 of `rollbar/__init__.py`:

```
from werkzeug.wrappers import Request as WerkzeugRequest
```

And line 1004:

```
    if WerkzeugRequest and isinstance(request, WerkzeugRequest):
        return _build_werkzeug_request_data(request)
```

Basically, rollbar is doing `isinstance` on `Request` rather than `BaseRequest`, so my custom request object fails the check and `_build_request_data` returns `None`.

I think the solution is fairly easy - either switch from `isinstance` to ducktyping, or check for `werkzeug.BaseRequest`.

**Comment by rivkahstandig3636** [comment]
Thanks for the feedback! @rokob What are your thoughts on this?

**Comment by rokob** [comment]
Reading what we do with werkzeug requests everything we look up is defined on BaseRequest and Requests are subclasses are BaseRequest so switching to BaseRequest will just work.

**Comment by rokob** [comment]
Fixed in #271
